packages/mathtype-html-integration-devkit/src/contentmanager.js

```diff
@@ -16,14 +16,15 @@ export default class ContentManager {
    * - The communication between the {@link ModalDialog} class and itself, if the content
    *   has been changed (hasChanges} method).
    * @constructs
-   * @param {Object} contentManagerAttributes - Object containing all attributes needed to
+   * @param {object} contentManagerAttributes - Object containing all attributes needed to
```

Comment:
There is a lot of linting changes; maybe those shouldn't be part of the PR since there's so many. In any case we can have another task/PR linting the devkit source files.